src/style.css

```diff
@@ -39,6 +39,8 @@ a {
 #root {
   overflow: auto;
   height: 100%;
+  position: relative;
```

Comment:
This creates a separate stacking context for the root, so that all of the elements within the root are displayed in order within their own root container. I added this so that the drop-shadow on the Top Bar would display above the Workspace Tabs (Data, Tools, Summary).